recipes/theseus/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
When I look at the tarball, it looks like the license file is included there as the `COPYING` file, so I think you can remove this file and use `license_file: COPYING` in the `meta.yaml`.